.github/CONTRIBUTING.md

```diff
@@ -67,4 +67,31 @@ The body should include the motivation for the change and contrast this with pre
 The footer should contain any information about **Breaking Changes** and is also the place to
 reference GitHub issues that this commit **Closes**.
 
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
\ No newline at end of file
+**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+
+## Changelog Guidelines
+All changes will be documented using a [CHANGELOG](https://github.com/rakutentech/android-miniapp/blob/master/CHANGELOG.md) by writing chronologically for each version with the following format:
+
+```
+Release Version (YYYY-MM-DD)
+SDK
+- <description>
+<BLANK LINE>
+Sample App
+- <description>
+```
+
+### Description
+A sentence describes the type of change with an information.
+
+```
+<bold><change type></bold>: <subject>
+```
+
+#### Change type
+* **Feature**: Added a new feature.
+* **Fixed**: When there is a hotfix or bugfix. Something has been upgraded could be mentioned too.
```

Comment:
The improvements can be contained either in Feature or in Improvements IMO, where it is more logical, considering their enhancing nature.
Whereas a fix is more of the type of making the existing thing work.

Reply:
Yeah, I am okay with use of "Improvement" keyword instead of "Fixed".